routes/skills.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const router = express.Router();
+const skills = require("../controllers/skills");
+const authenticate = require("../middlewares/authenticate");
+
+router.post("/:username", authenticate, skills.awardSkill);
+router.get("/", skills.fetchSkills);
+router.get("/:username", authenticate, skills.fetchUserSkills);
+router.get("/users/:skill", authenticate, skills.userWithSkill);
```

Comment:
This should be in the `users` not `skills`

Reply:
I was trying `/skills/:skill` but it didn't seem to work hence I wrote it this way...
But your point is right. It would be more obvious if routed through `/users`

Reply:
@rohan09-raj  
is it ok if that `/users/:skill` is `/users/skills/:skill`?
`/users/:skill` gets overlapped by `/users/:username` and tries to return user with `/:skill` as username

Reply:
Yeah I missed that, thanks for pointing it out. `/users/skills/:skill` is better